vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -41,21 +41,14 @@ if ($windowsSku -eq "2019") {
     $global:imagesToPull = @(
             "mcr.microsoft.com/windows/servercore:ltsc2019",
             "mcr.microsoft.com/windows/nanoserver:1809",
-            "mcr.microsoft.com/oss/kubernetes/pause:3.6-hotfix.20220114",
             "mcr.microsoft.com/oss/kubernetes/pause:3.9",
             # CSI. Owner: andyzhangx (Andy Zhang)
-            "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.5.0",
             "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.6.0",
-            "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.4.0",
             "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.5.0",
-            "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.26.2",
             "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.27.0",
-            "mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.24.0",
```

Comment:
keep `azurefile-csi:v1.24.0` since we are still using this version < AKS 1.26, and AKS 1.26 would use `azuredisk-csi:v1.26.2`

Reply:
Reverted a13a3d6a2 - chore: Delete old CSI images